packages/react-divider/src/components/Divider/useDividerStyles.ts

```diff
@@ -0,0 +1,432 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { DividerState } from './Divider.types';
+
+//#region Helper objects and functions
```

Comment:
Do we use #region elsewhere in the repo ?

Reply:
I'm not sure, I would recommend it as a valuable tool for coding, Maybe this can be brought up in a meeting?

Reply:
I have removed it for now since it wasn't wrapping any code from a refactor